.travis.yml

```diff
@@ -1,12 +1,19 @@
 language: node_js
 node_js:
-  - 0.10
+- 0.1
 before_script:
-  - npm install -g grunt-cli
-  - "phantomjs --version"
+- npm install -g grunt-cli
+- phantomjs --version
 notifications:
   email:
-    - sam@sceditor.com
```

Comment:
I did that for my personal builds off of my fork - should not have been included in pull request.  I'll revert that change.